optuna/_hypervolume/wfg.py

```diff
@@ -56,24 +57,37 @@ def compute_hypervolume(
         We check that each objective is to be minimized. Transform objective values that are
         to be maximized before calling this class's ``compute`` method.
 
+    Args:
+        loss_vals:
+            An array of loss value vectors to calculate the hypervolume.
+        reference_point:
+            The reference point used to calculate the hypervolume.
+        assume_pareto:
+            Whether to assume the Pareto optimality to ``loss_vals``.
+            In other words, if ``True``, none of loss vectors are dominated by another.
+
+    Returns:
+        hypervolume:
+            The hypervolume of the given arguments.
```

Comment:
I think this nesting is not necessary.

```suggestion
        The hypervolume of the given arguments.
```